lib/improver/tests/test_ensemble_calibration_EnsembleReordering.py

```diff
@@ -1,320 +0,0 @@
-        calibrated_cube.data = calibrated_data
-
-        plugin = Plugin()
-        result = plugin.rank_ecc(calibrated_cube, raw_cube)
-
-        result.transpose([1, 0, 2])
-
-        self.assertArrayAlmostEqual(result.data, result_data)
-
-    def test_3d_cube_tied_values(self):
-        """
-        Test that the plugin returns the correct cube data for a
-        3d input cube, when there are tied values witin the
-        raw ensemble members. As there are two possible options for the
-        result data, as the tie is decided randomly, both possible result
-        data options are checked.
-        """
-        raw_data = np.array(
-            [[[1, 1]],
-             [[3, 2]],
-             [[2, 2]]])
-
-        calibrated_data = np.array(
-            [[[1, 1]],
-             [[2, 2]],
-             [[3, 3]]])
-
-        # Reordering of the calibrated_data array to match
-        # the raw_data ordering
-        result_data_first = np.array(
-            [[[1, 1]],
-             [[3, 2]],
-             [[2, 3]]])
-
-        result_data_second = np.array(
-            [[[1, 1]],
-             [[3, 3]],
-             [[2, 2]]])
-
-        cube = self.cube.copy()
-        cube = cube[:, :, :2, 0]
-
-        raw_cube = cube.copy()
-        raw_cube.data = raw_data
-
-        calibrated_cube = cube.copy()
-        calibrated_cube.data = calibrated_data
-        plugin = Plugin()
-        result = plugin.rank_ecc(calibrated_cube, raw_cube)
-        result.transpose([1, 0, 2])
-
-        err_count = 0
-        try:
-            self.assertArrayAlmostEqual(result.data, result_data_first)
-        except Exception as err1:
-            err_count += 1
-
-        try:
-            self.assertArrayAlmostEqual(result.data, result_data_second)
-        except Exception as err2:
-            err_count += 1
-
-        if err_count == 2:
-            raise ValueError("Exceptions raised by both accepted forms of the "
-                             "calibrated data. {} {}".format(err1, err2))
-
-    def test_2d_cube(self):
-        """
-        Test that the plugin returns the correct cube data for a
-        2d input cube.
-        """
-        raw_data = np.array([[3],
-                             [2],
-                             [1]])
-
-        calibrated_data = np.array([[1],
-                                    [2],
-                                    [3]])
-
-        result_data = np.array([[3],
-                                [2],
-                                [1]])
-
-        cube = self.cube.copy()
-        cube = cube[:, :, 0, 0]
-        raw_cube = cube.copy()
-        raw_cube.data = raw_data
-        calibrated_cube = cube.copy()
-        calibrated_cube.data = calibrated_data
-
-        plugin = Plugin()
-        result = plugin.rank_ecc(calibrated_cube, raw_cube)
-        result.transpose([1, 0])
-        self.assertArrayAlmostEqual(result.data, result_data)
-
-
-class Test_process(IrisTest):
```

Comment:
Copied to lib/improver/tests/test_ensemble_copula_coupling_EnsembleReordering.py